### DISCUSSION TITLE
MHIG_EU_AU_K1555 MU 0415 - Maps update needs 21000ee activation key

### DISCUSSION THREAD

**Discussion by Misterious30** [Mr-MIBonk/M.I.B._More-Incredible-Bash#550] [Q&A]
Hi, firstly I really appreciate anyone taking their time out to respond to this post.  I have updated the firmware on my Audi A3 2016 to the one in the title.  I have the 210001e as the activation key, therefore unable to get maps past 2018 i think!

I was following mr-fix on youtube and it mentioned about patching the unit first to get the code.  However on https://mibsolution.one/ M.I.B folder there is a folder containing patches, this folder does not contain the firmware for my unit. 

What are my options? Any help would  be hugely appreciated thanks :)

**Comment by Misterious30** [comment]
Update: I have updated the GEM menu to 4.12 using tag v3.7.1 from this repo.  However, when I press PATCH now in the patch aio menu it does not do anything on the screen. its blank with 3 dots and I waited 10minutes

**Comment by Misterious30** [comment]
after running a backup which does not seem to say complete. I run the patch but nothing happens it stays blank...any ideas? I am running v3.7.1 M.I.B

![PXL_20240920_210435450 MP](https://github.com/user-attachments/assets/6f87a0cd-61d8-464f-a996-b460aae9d5b9)



![PXL_20240920_210602926 MP](https://github.com/user-attachments/assets/1cd04868-0ba8-43fd-894b-1a286e11e8a0)

**Reply by Fallemo** [reply]
Hey, how do you access the Green Engineering Menu? :D

**Reply by vaquero** [reply]
Hi.
In this post the partner explains it to you how activate the green menu:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/536#discussioncomment-12969768

and in this other post they explain the keys for each type of Audi model:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/622

**Comment by kamilcwalk94-blip** [comment]
AUDI A3 E-Tron 2015, Everywhere they say that you need to update the firmware, but you can't download it anywhere

<img width="1500" height="1125" alt="4ff3b911-d201-4615-8215-d9a1474b1580" src="https://github.com/user-attachments/assets/a8cccba2-12ce-48cd-a4a5-0eb1366ff368" />
<img width="1500" height="1125" alt="5950b8a2-4dcf-4bff-9d72-2feb36ee5c53" src="https://github.com/user-attachments/assets/5aba3171-4cd8-49ee-bc9d-d829f2df941a" />
<img width="1600" height="900" alt="WhatsApp Image 2026-04-25 at 18 23 18" src="https://github.com/user-attachments/assets/7500eb75-2767-4bd4-99a3-73b3d9e29dfb" />
